.circleci/config.yml

```diff
@@ -107,7 +107,9 @@ commands:
           command: |
             if [ ${CIRCLE_BRANCH} != "" ] &&
               [ ${CIRCLE_BRANCH} != "master" ] &&
-              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep -qvE '(.md$)|(.pdf$)'; then
+              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) \
```

Comment:
Have you tested what happens when we make a PR against a non-master branch? I've occasionally felt the need to do this (e.g. when lots of reviewers are on vacation). Do we just not run CircleCI jobs?